### DISCUSSION TITLE
Cannot log in to wikijs after deleting the jwt configuration

### DISCUSSION THREAD

**Discussion by erowz** [requarks/wiki#4713] [Help / Questions]
I deleted the content under the JWT configuration and saved it,but now I can’t log in..(it always shows logging in), how can I restore it? or how to export the backup？

**Comment by TJusczak** [comment]
I ran into the same issue. However I didn't delete that field. I just changed it from wiki.js to my url.

I fix it by going to my root wiki folder and opened this file /wiki/server/app/data.yml
This file does say not to change anything. But the Auth > Audience field still read 'urn:wiki.js'
I change it to 'urn:myweb.url', saved, and the login page worked right away.

Hopefully nothing else broke by changing this file. But so far everything looks good.

**Reply by Sergeus** [reply]
I had the exact same problem (changed the jwt url to my site url and then login stopped working) and this fixed it, thank you! Though I had to restart the container after modifying the file.

**Reply by Cmdr3nder** [reply]
I was able to make this same override by adding the following to my /etc/wiki-js/config.yml that overrides the defaults found in /wiki/server/app/data.yml. This should safely survive upgrades to wiki-js in the future.

```yml
auth:
  audience: 'urn:myweb.url'
```

Not sure why it is needed though. The system _should_ check the configuration stored in the db via the admin ui. Maybe the service needed a restart?
